R/gsutil.R

```diff
@@ -261,7 +271,7 @@ gsutil_cp <-
     )
 
     args <- c(
-        if (any(source_is_uri)) .gsutil_requesterpays_flag(source),
+        if (any(source_is_uri)) .gsutil_requesterpays_flag(source, destination),
```

Comment:
It seems from parsing the code that `.gsutil_is_uri()`, `gsutil_requesterpays()` and `gsutil_requesterpays_flag()` are (supposed to be) 'vectorized' in the `source` argument; is it sufficient to make no changes in those functions, and instead simply change line 265+ to
```
location <- c(source, destination)
location_is_uri <- .gsutil_is_uri(location)
...
args <- c(
    if (any(location_is_uri)) .gsutil_requesterpays_flag(location),
    ...
```